lib/samvera-persona.rb

```diff
@@ -1,20 +1,17 @@
 # requires all dependencies
 Gem.loaded_specs['samvera-persona'].dependencies.each do |d|
   require d.name unless d.type == :development
- end
+end
  
- require "samvera-persona/engine"
+require "samvera-persona/engine"
 
 module Samvera
   module Persona
-    class << self
-      mattr_accessor :soft_delete
-      self.soft_delete = true
+    mattr_accessor :soft_delete
```

Comment:
doesn't this change the scope? can you still call Samvera::Persona.soft_delete after this change or do you need to instantiate first?

Reply:
It worked for me in the test app and in Avalon.  I also added some tests for it: spec/lib/samvera-persona_spec.rb